### PR TITLE
Add issue and pull request templates

### PR DESCRIPTION
**Pull request by MatheusRich**
I added the templates just like the [issue](https://github.com/datasciencebr/serenata-de-amor/issues/303) asked for. Is that alright? 

I thought that maybe the "__"  should be only "_" to make the text _italic_. Anyway, I followed the issue instructions.

### CODE REVIEW

**Comment by jtemporal** [comment]
Hi @MatheusRich You are correct the `__` should be `_` to make the "help text" italic. Would you mind changing the double underscores to a single one? and while at it, I believe you might need to add a blank line between the **question** and the _help text_ to force the line break on the markdown

Thanks for this PR  😊

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+**What is the problem?** 
+__Try to give as many details about the bug/new feature/enhancement as possible, but keep it friendly for new comers__
```
Comment (jtemporal) [inline .github/ISSUE_TEMPLATE.md]: change the `__` to `_`

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+**What is the problem?**
```
Comment (jtemporal) [inline .github/ISSUE_TEMPLATE.md]: add a blank line to force the line break

**Comment by MatheusRich** [comment]
@jtemporal Done.

**Comment by jtemporal** [comment]
thanks @MatheusRich

**Pull request merged**